guides/Errors.md

```diff
@@ -1,17 +1,128 @@
 # Errors
+Error messages are a fundamental component of Wix's APIs, providing critical information and guidance when something goes wrong. 
+Most of Wix's errors follow a standardized format, as described below.
 
-When you call an API, the response you'll receive will include a status code. If something went wrong, you'll receive an error code that defines the type of error that occurred. 
+## Error properties
+| Property | Description | 
+| :-------------- | :------- |  
+| Status code | See the HTTP status code table below. |
```

Comment:
Thinking ahead...Is this not a property in a JSON or something like that when returned by the SDK? `code` or something like that?

Reply:
ooh, good point, I'm not sure what errors look like in the SDK. I'll check
(but in the meantime, I'll merge this to the REST ref as is)